components.json

```diff
@@ -1346,6 +1346,16 @@
 			"require": "markup",
 			"owner": "Golmote"
 		},
+		"wolfram": {
+			"title": "Wolfram language",
+			"alias": ["mathematica", "nb", "wl"],
+			"aliasTitles": {
+				"mathematica": "Mathematica",
+				"wl": "Wolfram Language",
```

Comment:
It's the same as the title of `wolfram`, so this can be omitted.

Reply:
But the L is capitalized?